src/components/Page.js

```diff
@@ -22,9 +23,16 @@ const Content = styled.div`
 
 const Page = ({ children }) => {
   const [showMobileSidebar, setShowMobileSidebar] = useState(false)
+  const [isJudgingEnabled, setIsJudgingEnabled] = useState(false)
+
+  useEffect(() => {
+    const unsubscribe = isJudgingOpen(setIsJudgingEnabled)
```

Comment:
why the name change from isJudgingOpen to isJudgingDisabled?

Reply:
idk its overlapping with the firebase utility function name